quote-book/tests/common/mod.rs

```diff
@@ -298,3 +300,68 @@ pub fn get_quotes_filtering_works(quote_book: &impl QuoteBook) {
         ]
     );
 }
+
+/// Test that get_quote_ids works.
+pub fn get_quote_ids_works(quote_book: &impl QuoteBook) {
+    let pair1 = pair();
+    let pair2 = Pair {
+        base_token_id: TokenId::from(10),
+        counter_token_id: TokenId::from(2),
+    };
+    let pair3 = Pair {
+        base_token_id: TokenId::from(20),
+        counter_token_id: TokenId::from(2),
+    };
+    let mut rng: StdRng = SeedableRng::from_seed([1u8; 32]);
+
+    let sci1 = create_sci(&pair1, 100, 1000, &mut rng);
```

Comment:
Nit: I found it a little surprising that sci2 was created using pair1 and sci3 was created using pair2. Might be slightly easier to read as sci2_pair1 or something.